unitex/src/fr/umlv/unitex/leximir/helper/DelasHelper.java

```diff
@@ -80,13 +81,16 @@ else if(dic!=null){
             //String path = Utils.getValueXml("pathDelas")+"/"+dela;
             String path="";
             if(allDelas)
```

Comment:
Braces are used with if, else, for, do and while statements, even when the body is empty or contains only a single statement [see here](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used)